packages/core/src/browser/preferences/preference-contribution.ts

```diff
@@ -224,6 +224,10 @@ export class PreferenceSchemaProvider extends PreferenceProvider {
 
     protected readonly unsupportedPreferences = new Set<string>();
     validate(name: string, value: any): boolean {
+        const overridden = this.overriddenPreferenceName(name);
```

Comment:
@a1994846931931 Could you elaborate why it is necessary? An idea was that a client should decide whether overridden preference name is to use or not. i.e it already happens here: https://github.com/theia-ide/theia/pull/6090/files#diff-ffb43b06d873f9f07d3ec2f4c762c96bR71 and it is not required here: https://github.com/theia-ide/theia/pull/6090/files#diff-ffb43b06d873f9f07d3ec2f4c762c96bR87 (returning overridden  value here is bogus)

Reply:
@akosyakov Oh, I see. The reason why I did this is that I got a lot of errors showing overridden preference name invalid.

So instead of getting overridden name for all validated preference, it should be done in the proxy, right?

Reply:
@a1994846931931 wait maybe i'm wrong, could you help me to reproduce it?

Reply:
@akosyakov Yes. So after removed line 227-230 in `preference-contribution.ts`, I ended in getting a great number of errors like
```
root WARN "[jsonc].editor.renderIndentGuides" preference is not supported
root WARN "[jsonc].editor.renderLineHighlight" preference is not supported
root WARN "[jsonc].editor.useTabStops" preference is not supported
root WARN "[jsonc].editor.colorDecorators" preference is not supported
root WARN "[jsonc].editor.highlightActiveIndentGuide" preference is not supported
root WARN "[jsonc].editor.iconsInSuggestions" preference is not supported
root WARN "[jsonc].editor.showUnused" preference is not supported
root WARN "[jsonc].editor.scrollBeyondLastColumn" preference is not supported
root WARN "[jsonc].editor.suggestSelection" preference is not supported
root WARN "[jsonc].editor.fontWeight" preference is not supported
root WARN "[jsonc].diffEditor.renderSideBySide" preference is not supported
```